### PR TITLE
add google analytics

### PR DESCRIPTION
**Pull request by Shyam-Chen**

### CODE REVIEW

**Comment by tarlepp** [comment]
Hmm, maybe some individual gulp task should take of case of this ? eg. using https://www.npmjs.com/package/gulp-ga

**Comment by mgechev** [comment]
It'll be better if the ID is configurable from project.config.ts. You can use the underscore (lodash) template syntax.

File: src/client/index.html [hunk 1]
```diff
@@ -16,6 +16,15 @@
 
   <sd-app>Loading...</sd-app>
 
+  <script defer>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+    ga('create', 'UA-XXXXXXXX-X', 'auto');
```
Comment (mgechev) [inline src/client/index.html]: Use interpolation with lodash template syntax and move the ID to config.

Comment (Shyam-Chen) [inline src/client/index.html]: done

File: tools/config/seed.config.ts [hunk 2]
```diff
@@ -147,6 +147,12 @@ export class SeedConfig {
   APP_TITLE = 'Welcome to angular-seed!';
 
   /**
+   * Tracking ID.
+   * @type {string}
+   */
+  Google_Analytics_ID = 'UA-XXXXXXXX-X';
```
Comment (mgechev) [inline tools/config/seed.config.ts]: Let's make it uppercase to be consistent with the other configuration properties.

**Comment by mgechev** [comment]
I added a comment. Let's change `Google_Analytics_ID` to `GOOGLE_ANALYTICS_ID`.

**Comment by mo-ba** [comment]
pls comment it out, i think its a bad idea, i always have to delete those lines

**Comment by mo-ba** [comment]
had to delete it 3 times in the last month

**Comment by Shyam-Chen** [comment]
If you do not use it can be ignored directly.

**Comment by mo-ba** [comment]
thats not true, it makes at least two http requests to google servers by default:
```
:authority:www.google-analytics.com
:method:GET
:path:/analytics.js
:scheme:https
accept:*/*
accept-encoding:gzip, deflate, sdch, br
accept-language:en-US,en;q=0.8,de;q=0.6
cache-control:no-cache
pragma:no-cache
referer:http://localhost:5555/
user-agent: XXXXXX
```
and 
`https://www.google-analytics.com/collect?XXXXXX`

**Pull request merged**